pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/export/DataExporter.java

```diff
@@ -638,10 +649,26 @@ private void convertInfoToJSONAndExportToES(Handle handle,
                     .doc(value, XContentType.JSON)
                     .docAsUpsert(true);
             bulkRequest.add(updateRequest);
+
+            if (isOsteoParticipantStructuredIndex) {
+                UpdateRequest updateRequestOS2 = new UpdateRequest()
+                        .index(CMI_OSTEO2_INDEX)
+                        .id(key)
+                        .doc(value, XContentType.JSON)
+                        .docAsUpsert(true);
+                finalBulkRequestOS2.add(updateRequestOS2);
+            }
         });
 
-        BulkResponse bulkResponse = esClient.bulk(bulkRequest, RequestOptions.DEFAULT);
+        if (finalBulkRequestOS2 != null) {
+            log.info("exporting {} participant records to index participants_structured.cmi.cmi-osteo2", participantRecords.size());
```

Comment:
Cleaner if 
`log.info("exporting {} participant records to index {}", participantRecords.size(), CMI_OSTEO2_INDEX);`